dev-db/influxdb/influxdb-2.7.3.ebuild

```diff
@@ -0,0 +1,284 @@
+	tinyvec_macros@0.1.1
+	typed-arena@2.0.2
+	unicase@2.6.0
+	unicode-bidi@0.3.13
+	unicode-ident@1.0.8
+	unicode-normalization@0.1.22
+	unicode-segmentation@1.10.1
+	unicode-width@0.1.10
+	url@2.3.1
+	vcpkg@0.2.15
+	vec_map@0.8.2
+	version_check@0.9.4
+	walkdir@2.3.3
+	wasi@0.11.0+wasi-snapshot-preview1
+	wasm-bindgen-backend@0.2.84
+	wasm-bindgen-macro-support@0.2.84
+	wasm-bindgen-macro@0.2.84
+	wasm-bindgen-shared@0.2.84
+	wasm-bindgen@0.2.84
+	web-sys@0.3.61
+	winapi-i686-pc-windows-gnu@0.4.0
+	winapi-util@0.1.5
+	winapi-x86_64-pc-windows-gnu@0.4.0
+	winapi@0.3.9
+	windows-sys@0.45.0
+	windows-sys@0.48.0
+	windows-targets@0.42.2
+	windows-targets@0.48.0
+	windows@0.48.0
+	windows_aarch64_gnullvm@0.42.2
+	windows_aarch64_gnullvm@0.48.0
+	windows_aarch64_msvc@0.42.2
+	windows_aarch64_msvc@0.48.0
+	windows_i686_gnu@0.42.2
+	windows_i686_gnu@0.48.0
+	windows_i686_msvc@0.42.2
+	windows_i686_msvc@0.48.0
+	windows_x86_64_gnu@0.42.2
+	windows_x86_64_gnu@0.48.0
+	windows_x86_64_gnullvm@0.42.2
+	windows_x86_64_gnullvm@0.48.0
+	windows_x86_64_msvc@0.42.2
+	windows_x86_64_msvc@0.48.0
+	yansi@0.5.1
+"
+
+inherit cargo go-module systemd
+
+DESCRIPTION="Scalable datastore for metrics, events, and real-time analytics"
+HOMEPAGE="https://www.influxdata.com"
+
+FLUX_PV="0.194.3"
+
+SRC_URI="https://github.com/influxdata/influxdb/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://github.com/influxdata/ui/releases/download/OSS-v2.7.1/build.tar.gz -> ${P}-assets.tar.gz"
+SRC_URI+=" https://gentoo.kropotkin.rocks/go-pkgs/${P}-deps.tar.xz"
+SRC_URI+=" $(cargo_crate_uris)"
+
+LICENSE="Apache-2.0 BSD BSD-2 EPL-2.0 ISC MIT MPL-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="test"
+RESTRICT="!test? ( test )"
+
+BDEPEND="virtual/pkgconfig"
+
+COMMON_DEPEND="
+	acct-group/influxdb
+	acct-user/influxdb
+"
+
+DEPEND="${COMMON_DEPEND}"
+RDEPEND="${COMMON_DEPEND}"
+
+src_unpack() {
+	cargo_src_unpack
+}
+
+src_prepare() {
+	default
+
+	local data_dir
+	data_dir="${S}/static/data"
+	mkdir "${data_dir}" || die
+	mv "${WORKDIR}/build" "${data_dir}" || die
+}
+
+src_compile() {
+	mv "${WORKDIR}/go-mod" "${WORKDIR}/go-mod-tmp" || die
+	mv "${WORKDIR}/go-mod-tmp/github.com/influxdata/pkg-config@v0.2.11/go-mod" "${WORKDIR}/go-mod" || die
+	cd "${WORKDIR}"/go-mod-tmp/github.com/influxdata/pkg-config* || die
+	ego build .
+	mv "${WORKDIR}/go-mod" "${WORKDIR}/go-mod-tmp/github.com/influxdata/pkg-config@v0.2.11" || die
+	mv "${WORKDIR}/go-mod-tmp" "${WORKDIR}/go-mod" || die
+
+	cd "${WORKDIR}/go-mod/github.com/influxdata/flux@v${FLUX_PV}/libflux" || die
+	cargo_src_compile
+
+	cd "${S}" || die
+
+	export PKG_CONFIG="${WORKDIR}/go-mod/github.com/influxdata/pkg-config@v0.2.11/pkg-config"
+	ego generate ./static
+	GOBIN="${S}/bin" \
+		ego install \
+			-tags 'assets,noasm,sqlite_json,sqlite_foreign_keys' \
+			-ldflags="-X main.version=${PV}" \
+		./...
+}
+
+src_test() {
+	ego test ./tests
+}
+
+src_install() {
+	dobin bin/influx*
+	dodoc *.md
+	cd .circleci/scripts/package/influxdb2/fs
```

Comment:
missing `|| die`

Reply:
Thanks. I also removed the two dodir which seems to be redundant.